### PR TITLE
Correct wrong data structure description of MBIM_MS_SET_TRANSMISSION_STATUS_INFO 

### PR DESCRIPTION
**Pull request by zarcen**
A couple typos in the description. Correct them to meet the Parameters table.

### CODE REVIEW

File: windows-driver-docs-pr/network/mb-sar-platform-support.md [hunk 1]
```diff
@@ -156,11 +156,11 @@ OEMs that use this command should be aware of the potential power impact as the
 
 #### Query
 
-The InformationBuffer on MBIM_COMMAND_MSG is not used. MBIM_MS_SET_TRANSMISSION_STATUS_INFO is returned in the InformationBuffer of MBIM_COMMAND_DONE.
+The InformationBuffer on MBIM_COMMAND_MSG is not used. MBIM_MS_TRANSMISSION_STATUS_INFO is returned in the InformationBuffer of MBIM_COMMAND_DONE.
 
 #### Set
 
-The InformationBuffer on MBIM_COMMAND_MSG contains MBIM_MS_SET_TRANSMISSION_STATUS. MBIM_MS_SET_TRANSIMISSION_STATUS _INFO is returned in the InformationBuffer of MBIM_COMMAND_DONE.
+The InformationBuffer on MBIM_COMMAND_MSG contains MBIM_MS_SET_TRANSMISSION_STATUS. MBIM_MS_TRANSMISSION_STATUS_INFO _INFO is returned in the InformationBuffer of MBIM_COMMAND_DONE.
```
Comment (tenglu83) [inline windows-driver-docs-pr/network/mb-sar-platform-support.md:163]: This "_INFO" looks like a typo.

Comment (zarcen) [reply windows-driver-docs-pr/network/mb-sar-platform-support.md]: Thanks. It is. Resolved it in patch-2.

**Comment by zarcen** [comment]
@billlattimer @EliotSeattle Could someone take a look at the change?

**Comment by EliotSeattle** [comment]
Thanks, @zarcen. Sorry for the delay.

**Pull request merged**